tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -258,7 +256,7 @@ int main(int argc, char **argv)
 
             /* Calculate extension list offset. Extension list should be last. */
             const uint32_t extension_list_offset = sizeof(uint32_t) /* max_early_data_size */
-                    + sizeof(uint16_t) /* size of extension */
+                    + sizeof(uint16_t)                              /* size of extension */
                     + sizeof(uint16_t) /* type of extension */;
```

Comment:
Why did it indent the 259 comment but not the 260 comment?

Reply:
Its the fact the semicolon is after the comment. Switch that around and it works.